File: Day3/factorial.cpp
#include <iostream>
using namespace std;

int main()
{
    int i,num ,fact=1;
    cout<<"Enter a number";
    cin>>num;
    for(i=2;i<=num;i++)
    {
    fact=fact*i;
    

    }
    cout<<"the factorial is "<<fact<<endl;
}

File: WPT/WBT Day 20/src/Calculator/Factorial.js
import React from 'react';

export default function Factorial({ number }) {
  const factorial = (n) => {
    if (n < 0) return 'Invalid';
    let fact = 1;
    for (let i = 1; i <= n; i++) {
      fact *= i;
    }
    return fact;
  };

  return <p>Factorial of {number} is {factorial(number)}</p>;
}
